utils/math.js

```diff
@@ -0,0 +1,25 @@
+// math.js
+
+// Function to generate a random integer within a specified range
```

Comment:
@gblotta-xoor Here we can use `JSDoc`[^1]. Would you mind converting this comment to `JSDoc`?

[^1]: https://jsdoc.app/

Reply:
@jaymdq https://github.com/jaymdq/code-review-process-workshop/pull/3/commits/03bee22f8f0e1f6ee0dbe0581a029051932d4ffc